src/scanner-engine.ts

```diff
@@ -106,6 +109,20 @@ export async function runScannerEngine(
     '-jar',
     scannerEnginePath,
   ];
+
+  // If debugging with dumpToFile, write the properties to a file and exit
+  const dumpToFile = properties[ScannerProperty.SonarScannerInternalDumpToFile];
+  if (dumpToFile) {
+    const data = {
+      propertiesJSON,
+      javaBinPath,
+      scannerEnginePath,
+      args,
+    };
+    log(LogLevel.DEBUG, 'Dumping data to file', data);
```

Comment:
- `Dumping data to file and exiting`
- Change to INFO log level
- Drop `data` from stdout